website/docs/user_guides/io_chaos.md

```diff
@@ -94,18 +17,17 @@ apiVersion: chaos-mesh.org/v1alpha1
 kind: IoChaos
 metadata:
   name: io-delay-example
-  namespace: chaos-testing
 spec:
-  action: mixed
+  action: latency
   mode: one
-  duration: "400s"
-  path: ""
   selector:
     labelSelectors:
-      "app.kubernetes.io/component": "tikv"
-  layer: "fs"
-  percent: "50"
-  delay: "1ms"
+      app: etcd
+  volumePath: /var/run/etcd
```

Comment:
The description of `volumePath` is missing below.

Reply:
Fixed